src/extension/options-page/index.tsx

```diff
@@ -135,65 +134,64 @@ function DashboardUI() {
-                    )}
-                </div>
-                {xsMatched ? null : (
-                    <footer className={classes.footer}>
-                        <FooterLine />
-                    </footer>
+        <div className={classes.root}>
+            <div className={classes.container}>
+                {loading ? (
+                    <Box className={classes.suspend}>
+                        <CircularProgress />
+                    </Box>
+                ) : (
+                    <>
+                        {xsMatched ? null : drawer}
+                        <Switch>
+                            {webpackEnv.perferResponsiveTarget === 'xs' ? (
+                                <Route path={DashboardRoute.Nav} component={nav} />
+                            ) : null}
+                            <Route path={DashboardRoute.Personas} component={DashboardPersonasRouter} />
+                            <Route path={DashboardRoute.Wallets} component={DashboardWalletsRouter} />
+                            <Route path={DashboardRoute.Contacts} component={DashboardContactsRouter} />
+                            <Route path={DashboardRoute.Settings} component={DashboardSettingsRouter} />
+                            <Route path={DashboardRoute.Setup} component={DashboardSetupRouter} />
+                            {/* // TODO: this page should be boardless */}
+                            <Route path={DashboardRoute.RequestPermission} component={RequestPermissionPage} />
+                            <Redirect
+                                path="*"
+                                to={
+                                    webpackEnv.perferResponsiveTarget === 'xs'
+                                        ? DashboardRoute.Nav
+                                        : DashboardRoute.Personas
+                                }
+                            />
+                        </Switch>
+                    </>
                 )}
             </div>
-        </DashboardBlurContextUI>
+            {xsMatched ? null : (
+                <footer className={classes.footer}>
+                    <FooterLine />
+                </footer>
+            )}
+        </div>
     )
 }
 
 export default function Dashboard() {
     const preferDarkScheme = useMediaQuery('(prefers-color-scheme: dark)')
     const appearance = useValueRef(appearanceSettings)
+    const theme =
+        (preferDarkScheme && appearance === Appearance.default) || appearance === Appearance.dark
+            ? MaskbookDarkTheme
+            : MaskbookLightTheme
     return (
         <I18nextProvider i18n={i18nNextInstance}>
-            <ThemeProvider
-                theme={
-                    (preferDarkScheme && appearance === Appearance.default) || appearance === Appearance.dark
-                        ? MaskbookDarkTheme
-                        : MaskbookLightTheme
-                }>
+            <ThemeProvider theme={Object.create(theme)}>
```

Comment:
Why use Object.create?

Reply:
This one isn't necessary. Just follow the change in the popup page. I will undo it.

Reply:
It leads to a new theme every-render

Reply:
That's true. I'm removing it.